From 9b2be6dec7f8225e5c9eb1bf2e050bf364f13f9c Mon Sep 17 00:00:00 2001
From: John Jacques <john.jacques@intel.com>
Date: Mon, 27 Apr 2015 10:20:24 -0500
Subject: [PATCH 023/131] arch/arm64: Axxia Device Tree Update

The interrupt numbers for general purpose DMA were off by 32.

Signed-off-by: John Jacques <john.jacques@intel.com>
[Original patch is from linux-yocto-4.1 axxia/base branch.]
Signed-off-by: Yanjiang Jin <yanjiang.jin@windriver.com>
---
 arch/arm64/boot/dts/intel/axm56xx.dtsi | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm64/boot/dts/intel/axm56xx.dtsi b/arch/arm64/boot/dts/intel/axm56xx.dtsi
index 3b46ea0..99287f2 100644
--- a/arch/arm64/boot/dts/intel/axm56xx.dtsi
+++ b/arch/arm64/boot/dts/intel/axm56xx.dtsi
@@ -142,7 +142,7 @@
 		gpdma0: gpdma@8004120000 {
                         compatible = "lsi,dma32";
                         reg = <0x80 0x04120000 0x10000>;
-			interrupts = <GIC_SPI 102 IRQ_TYPE_LEVEL_HIGH>,
+			interrupts = <GIC_SPI 70 IRQ_TYPE_LEVEL_HIGH>,
 					<GIC_SPI 71 IRQ_TYPE_LEVEL_HIGH>;
                         channel0 {
                                 interrupts = <GIC_SPI 72 IRQ_TYPE_LEVEL_HIGH>;
@@ -156,7 +156,7 @@
                 gpdma1: gpdma@8004130000 {
                         compatible = "lsi,dma32";
                         reg = <0x80 0x04130000 0x10000>;
-			interrupts = <GIC_SPI 106 IRQ_TYPE_LEVEL_HIGH>,
+			interrupts = <GIC_SPI 74 IRQ_TYPE_LEVEL_HIGH>,
 					<GIC_SPI 75 IRQ_TYPE_LEVEL_HIGH>;
 			status = "disabled";
                         channel0 {
-- 
2.8.1

